data/index.html

```diff
@@ -11,23 +11,23 @@
     <div class="content">
       <div class="title">DIY Air Quality Monitor</div>
       <div id="current" class="aqi-display">
-        <div class="^COLOR-CURRENT^">
-          <div class="aqi-value">^AQI-CURRENT^</div>
+        <div class="^COLOR-CURRENT^" id="bg-current">
+          <div class="aqi-value" id="value-current">^AQI-CURRENT^</div>
         </div>
```

Comment:
All these changes main to `index.html` need to also be made to `index_bme680.html` as that is the version of the index HTML that is used with the BME680 sensor is attached. Better yet, maybe consolidate everything to one index HTML file and have a method that strips out the BME680 divs if the sensor is not present.

Reply:
I did as you suggested and consolidated them both into a single file, which should only show the environment data if the sensor is available.